app/src/main/java/com/example/nuberjam/data/source/remote/service/ApiConfig.kt

```diff
@@ -1,4 +1,26 @@
 package com.example.nuberjam.data.source.remote.service
 
-class ApiConfig {
+import com.example.nuberjam.BuildConfig
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object ApiConfig {
+    const val BASE_URL = "http://nuberjam.orgfree.com/api/"
```

Comment:
I think we need to separate this into a new object class like `Constant.kt`. This is used when we need to use the build variant feature.

Reply:
okay